src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/IDE/VSSolutionManager.cs

```diff
@@ -514,14 +517,14 @@ private void SolutionSaveAs_AfterExecute(string Guid, int ID, object CustomIn, o
                 NuGetUIThreadHelper.JoinableTaskFactory.Run(async delegate
                 {
                     await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
can remove this switch, since OnSolutionExistsAndFullyLoadedAsync will do the switch